add `qualified-do`

artemisSystem opened the pull request.

JordanMartinez commented on new-packages.json:
```diff
@@ -186,5 +186,6 @@
   "purescript-argparse-basic": "https://github.com/natefaubion/purescript-argparse-basic.git",
-  "purescript-web-page-visibility": "https://git.sr.ht/~toastal/purescript-web-page-visibility"
+  "purescript-web-page-visibility": "https://git.sr.ht/~toastal/purescript-web-page-visibility",
+  "purescript-qualified-do": "https://github.com/artemisSystem/purescript-qualified-do.git"
```
Is this a private repo? https://github.com/artemisSystem/purescript-qualified-do returns 404.

artemisSystem commented:
Whoops, i initially forgot the `purescript-` prefix when creating the repo. Renamed it now so that the link works. Also rebased to fix conflict with #393
